examples/tlc5947_chain.py

```diff
@@ -0,0 +1,78 @@
+# Simple demo of controlling a chain of several TLC5947 12-bit 24-channel PWM controllers.
+# Will update channel values to different PWM duty cycles.
+# Authors: Tony DiCola, Walter Haschka
+
+import board
+import busio
+import digitalio
+
+import adafruit_tlc5947
+
+# Define pins connected to the TLC5947
+SCK = board.SCK
+MOSI = board.MOSI
+LATCH = digitalio.DigitalInOut(board.D5)
+
+# Initialize SPI bus.
+spi = busio.SPI(clock=SCK, MOSI=MOSI)
+
+# Initialize TLC5947
+DRIVER_COUNT = 2		# change this to the number of drivers you have chained
+tlc5947 = adafruit_tlc5947.TLC5947(spi, LATCH, num_drivers=DRIVER_COUNT)
+# You can optionally disable auto_write which allows you to control when
+# channel state is written to the chip.  Normally auto_write is true and
+# will automatically write out changes as soon as they happen to a channel, but
+# if you need more control or atomic updates of multiple channels then disable
+# and manually call write as shown below.
+#tlc5947 = adafruit_tlc5947.TLC5947(spi, LATCH, num_drivers=DRIVER_COUNT, auto_write=False)
+
+# There are two ways to set channel PWM values.  The first is by getting
+# a PWMOut object that acts like the built-in PWMOut and can be used anywhere
+# it is used in your code.  Change the duty_cycle property to a 16-bit value
+# (note this is NOT the 12-bit value supported by the chip natively) and the
+# PWM channel will be updated.
+
+# With one RGB LED hooked up to pins 0, 1, and 2, AND
+# with another RGB LED hooked up to pins 21, 22 and 23 of the last chained driver,
+# cycle the red, green, and blue pins of LED one up and down, and the other vice versa:
+
+redA = tlc5947.create_pwm_out(0)
+greenA = tlc5947.create_pwm_out(1)
+blueA = tlc5947.create_pwm_out(2)
+redZ = tlc5947.create_pwm_out(DRIVER_COUNT*24-3)
+greenZ = tlc5947.create_pwm_out(DRIVER_COUNT*24-2)
+blueZ = tlc5947.create_pwm_out(DRIVER_COUNT*24-1)
+
+step = 10
+start_pwm = 0
+end_pwm = 32767 # 50% (32767, or half of the maximum 65535):
+
+while True:
+    for (pinA, pinZ) in ((redA, redZ), (greenA, greenZ), (blueA, blueZ)):
+        # Brighten:
+        print("LED A up, LED Z down")
+        for pwm in range(start_pwm, end_pwm, step):
+            pinA.duty_cycle = pwm
+            pinZ.duty_cycle = end_pwm - pwm
+#            tlc5947.write()        # see NOTE below
+
+        # Dim:
+        print("LED A down, LED Z up")
+        for pwm in range(end_pwm, start_pwm, 0 - step):
+            pinA.duty_cycle = pwm
+            pinZ.duty_cycle = end_pwm - pwm
```

Comment:
just do the same with both LEDs:
```suggestion
            pinZ.duty_cycle = pwm
```